src/pages/console/formbuilder/overview.mdx

```diff
@@ -11,7 +11,7 @@ Forms are also extensible, allowing full form lifecycle management, and supporti
 
 The fastest and easiest way to use the Form Builder is to use forms generated from your data model. To auto-generate forms:
 
-1. Deploy a data model in Amplify Studio - [Learn more](/console/data/data-model/)
+1. Deploy a [Data model](/console/data/data-model/) in [Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app)
```

Comment:
I would add "In Amplify Studio..." to the beginning of this sentence, not the end